ci/script.sh

```diff
@@ -178,6 +178,14 @@ EOF
 
         rm -rf $td
     fi
+
+    cd crates
```

Comment:
> https://travis-ci.org/rust-embedded/cross/jobs/465538092#L3778
> ci/script.sh: line 178: cd: crates: No such file or directory

Should this be `pushd ci/crates/docker_context` and `popd`?

Also README caveats/gotchas section needs to be updated.